lib/Jejik/MT940/Account.php

```diff
@@ -40,7 +42,7 @@ class Account implements AccountInterface
      *
      * @return string
      */
-    public function getNumber()
+    public function getNumber(): ?string
```

Comment:
@return and @param is removed.

I must change the type from string to ?string, because the tests are failed.
In a future PR I will change the type to string and check the problems. But thats to big for one PR.